packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/ValidateModelConfigHandler.java

```diff
@@ -0,0 +1,46 @@
+package software.uncharted.terarium.hmiserver.service.tasks;
+
+import java.util.UUID;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import software.uncharted.terarium.hmiserver.models.task.TaskResponse;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class ValidateModelConfigHandler extends TaskResponseHandler {
+	public static final String NAME = "funman_task:validate_modelconfig";
+
+	@Override
+	public String getName() {
+		return NAME;
+	}
+
+	@Data
+	public static class Properties {
+		UUID simulationId;
+	}
+
+	@Override
+	public TaskResponse onSuccess(final TaskResponse resp) {
+		try {
+			final Properties props = resp.getAdditionalProperties(Properties.class);
+			final UUID simulationId = props.getSimulationId();
+
+			System.out.println("");
+			System.out.println("");
+			System.out.println("");
+			System.out.println("");
+			System.out.println("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! " + simulationId.toString());
+			System.out.println("");
+			System.out.println("");
+			System.out.println("");
+			System.out.println("");
+		} catch (final Exception e) {
+			throw new RuntimeException(e);
+		}
+		return resp;
```

Comment:
This can be in the `try {}`?

Reply:
I don't think it makes a difference here, nor will it make a difference once I filled it in with the S3/DB-update stuff in the next PR. 

So I guess the Q is, do we have a preferred stylistic choice?

Reply:
Is this entire `try` block needed? It looks like its just for the logging and this is just returning `resp`

Reply:
try/catch will be needed, this is just a bit of scaffolding for the next PR.